README.md

```diff
@@ -79,6 +77,7 @@ specific language governing permissions and limitations under the License.
 [l5d]: https://linkerd.io/
 [license-badge]: https://img.shields.io/github/license/linkerd/linkerd.svg
 [logo]: https://user-images.githubusercontent.com/240738/33589722-649152de-d92f-11e7-843a-b078ac889a39.png
+[roadmap]:https://conduit.io/roadmap
 [releases]: https://github.com/runconduit/conduit/releases
 [rust]: https://rust-lang.org/
 [twitter]: https://twitter.com/runconduit/
```

Comment:
The "twitter" link is no longer used in this document -- you can safely remove it.